Makefile

```
@@ -504,6 +499,59 @@ stats-download-kpi: ## Generate download KPI JSON for all published articles (op
+		$(if $(query),--query=$(query)) \
+		$(if $(filter 1,$(sync)),--sync)
+
+solr-worker: ## Continuously consume the Solr indexing/deletion queue (optional: limit=N time-limit=SECONDS memory-limit=SIZE)
+	# Prod: sudo -u $(CNTR_APP_USER) php $(CNTR_APP_DIR)/scripts/console.php solr:worker [--limit=N] [--time-limit=SECONDS] [--memory-limit=SIZE] [-q]
+	@$(DOCKER_COMPOSE) exec -u $(CNTR_APP_USER) -w $(CNTR_APP_DIR) $(CNTR_NAME_PHP) \
+		php scripts/console.php solr:worker \
+		$(if $(limit),--limit=$(limit)) \
+		$(if $(time-limit),--time-limit=$(time-limit)) \
+		$(if $(memory-limit),--memory-limit=$(memory-limit))
+
+solr-queue: ## Inspect/manage the Solr indexing queue (requires stats=1 or list-failed=1 or retry=ID; optional: limit=N)
```

checkmake (0.3.2)

[warning] Target body for "solr-queue" exceeds allowed length of 5 lines (13).

(maxbodylength)